woocommerce/woocommerce
# [Fix] Undefined array key 1, 2, 3 when using list to extract cookie values.

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:
- Check cookie parts length before checking if it is valid cookie.




### How to test the changes in this Pull Request:





1. Add a product to cart
2. Check woocommerce cookie in browser debug console and modify `wp_woocommerce_session_xxx` cookie in such a way that format is not valid anymore. Ex: add random value `abc`
3. Reload the page.
4. The cart should be empty and session is not initialized.
